### PR TITLE
Changes in PlayerSizeHandler

### PR DESCRIPTION
**Pull request by isitrita**
I moved logic of centering and resizing of player from js code to css part.

### CODE REVIEW

File: src/foundation-player.scss [hunk 1]
```diff
@@ -43,10 +43,13 @@ $fp-button-hover-scale: 1.2 !default;
...
 
+
 // Foundation player styles
 @include exports('foundation-player') {
   // Common styles for player parts
   .foundation-player {
+    display: table;
+    margin: 0 auto;
     .player {
       margin: 0 auto; // Center player
```
Comment (sashaegorov) [inline src/foundation-player.scss:54]: @isitrita which of two `margin: 0 auto;` is still required?

File: src/foundation-player.coffee [hunk 2]
```diff
@@ -210,15 +210,9 @@
       else
         console.error 'setPlayerSize: incorrect size argument'
         return false
-    # Update player elemant width
+    # Obsolete PlayerSizeHandler funct
```
Comment (sashaegorov) [inline src/foundation-player.coffee:219]: If it was removed, i think it should not be mentioned here.

File: src/foundation-player.scss (hunk 1 above)
Comment (isitrita) [inline src/foundation-player.scss:54]: Yes, it works correctly with margin: 0 auto in both places.

File: src/foundation-player.coffee [hunk 3]
```diff
@@ -202,10 +205,8 @@
     # Look and feel ============================================================
     # This method toggles player size
     togglePlayerSize: ->
-      switchToSize = if @currentPlayerSize == 'normal' then 'small' else 'normal'
-      switchClass @$wrapper, switchToSize, @currentPlayerSize
-      @setPlayerSizeHandler()
-      @currentPlayerSize = switchToSize
+      setPlayerSize('small') if @currentPlayerSize == 'normal'
+      setPlayerSize('normal') if @currentPlayerSize == 'small'
```
Comment (sashaegorov) [inline src/foundation-player.coffee:209]: You reading default value of `currentPlayerSize` here `setPlayerSize('normal') if @currentPlayerSize == 'small'`.
After switching size `currentPlayerSize` doesn't get updated. Right? Have you tested `currentPlayerSize` attribute value?

Comment (isitrita) [inline src/foundation-player.coffee:209]: `setPlayerSize()` updates currentPlayerSize.

File: src/foundation-player.scss (hunk 1 above)
Comment (sashaegorov) [inline src/foundation-player.scss:54]: Yep, that is my point, why do we need _both_? I'm trying to keep markup clean and DRY.
Can you test and delete one of them which is redundant?

File: src/foundation-player.coffee (hunk 3 above)
Comment (sashaegorov) [inline src/foundation-player.coffee:209]: `setPlayerSize` should be `@setPlayerSize`. Right?

File: src/foundation-player.coffee (hunk 2 above)
Comment (isitrita) [inline src/foundation-player.coffee:219]: Naming of function should be changed. I will add TODO for that.

File: src/foundation-player.scss (hunk 1 above)
Comment (isitrita) [inline src/foundation-player.scss:54]: Yes, for `.player` class `margin:0 auto` removes margin from parent classes. And for `foundation_player` it it is centering content.

**Pull request merged**